### DISCUSSION TITLE
rename_fields explanation please

### DISCUSSION THREAD

**Discussion by agorkiy** [vectordotdev/vector#12116] [Q&A] [answered]
Hi
I need to rename my field, `timestamp` to `@timestamp`. How this can me done?
I see that vectore added `rename_fields` function, but I don't see any examples of how to use it.
Thanks.

**Comment by spencergilbert** [comment, marked as answer]
`rename_fields` has been deprecated in favor of the more flexible [`remap`](https://vector.dev/docs/reference/configuration/transforms/remap/) - the VRL script to rename would be as simple as:

```coffeescript
.@timestamp = del(.timestamp)
```
